k8s/tests/manifests/tests.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: "keramik.3box.io/v1alpha1"
+kind: Network
+metadata:
+  name: migration-tests
+spec:
+  replicas: 2
+  ceramic:
+    ipfs:
+      rust: {}
+
+---
+kind: Job
```

Comment:
Does this job wait to run until the network is ready? Or is the idea that you don't apply this definition until the network is ready?

Reply:
That's exactly the thing I'm looking into next! I'd like for the tests to wait till the job is up though it shouldn't be hard to just wait and start the job by hand, which is what I've been doing to test.